tests/test_selector_jmespath.py

```diff
@@ -77,11 +82,65 @@ def test_html_has_json(self):
         """
         sel = Selector(text=html_text)
         self.assertEqual(
-            sel.xpath("//div/content/text()").jmespath("user[*].name").getall(),
+            sel.xpath("//div/content/text()")
+            .jmespath("user[*].name")
+            .getall(),
             ["A", "B", "C", "D"],
         )
         self.assertEqual(
             sel.xpath("//div/content").jmespath("user[*].name").getall(),
             ["A", "B", "C", "D"],
         )
         self.assertEqual(sel.xpath("//div/content").jmespath("total").get(), 4)
+
+    def test_jmespath_functions(self):
```

Comment:
I don’t think we need this test function, whether JMESPath functions work or not does not depend on Parsel, but on the jmespath PyPI package.